migrate-from-mysql-dumpling-files.md

```diff
@@ -6,7 +6,7 @@ aliases: ['/docs-cn/dev/migrate-from-mysql-mydumper-files/']
 
 # 从 MySQL SQL 文件迁移数据
 
-本文介绍如何使用 TiDB Lightning 从 MySQL SQL 文件迁移数据到 TiDB。关于如何生成 MySQL SQL 文件，可以参考 [Mydumper](/mydumper-overview.md) 或者 [Dumpling](/dumpling-overview.md) 文档。
+本文介绍如何使用 TiDB Lightning 从 MySQL SQL 文件迁移数据到 TiDB。关于如何生成 MySQL SQL 文件，可以参考 [Dumpling](/dumpling-overview.md) 文档。
```

Comment:
See [Line 25](https://github.com/pingcap/docs-cn/pull/4857/files#diff-ef12f61d3e9152c317c11d7cb270c8253babce2de8aadad0fb3fe510138f5ef6R25). It's still mudumper config.
)

Reply:
In lightning this config is named as mydumper. But we recommend using dumpling to export this.